public/app/percona/dbaas/components/DBCluster/XtraDB.service.ts

```diff
@@ -27,59 +26,48 @@ import {
 import { getComponentChange } from './DBCluster.service.utils';
 import { Operators } from './AddDBClusterModal/DBClusterBasicOptions/DBClusterBasicOptions.types';
 
-export const DBCLUSTER_STATUS_MAP = {
-  [DBClusterStatus.invalid]: 'XTRA_DB_CLUSTER_STATE_INVALID',
-  [DBClusterStatus.changing]: 'XTRA_DB_CLUSTER_STATE_CHANGING',
-  [DBClusterStatus.ready]: 'XTRA_DB_CLUSTER_STATE_READY',
-  [DBClusterStatus.failed]: 'XTRA_DB_CLUSTER_STATE_FAILED',
-  [DBClusterStatus.deleting]: 'XTRA_DB_CLUSTER_STATE_DELETING',
-  [DBClusterStatus.suspended]: 'XTRA_DB_CLUSTER_STATE_PAUSED',
-  [DBClusterStatus.upgrading]: 'XTRA_DB_CLUSTER_STATE_UPGRADING',
-  [DBClusterStatus.unknown]: 'XTRA_DB_CLUSTER_STATE_UNKNOWN',
-};
-
 export class XtraDBService extends DBClusterService {
-  getDBClusters(kubernetes: Kubernetes): Promise<DBClusterPayload> {
-    return apiManagement.post<any, Kubernetes>('/DBaaS/XtraDBClusters/List', kubernetes, true);
-  }
-
   addDBCluster(dbCluster: DBCluster): Promise<void | DBClusterPayload> {
-    return apiManagement.post<DBClusterPayload, any>('/DBaaS/XtraDBCluster/Create', toAPI(dbCluster));
+    return apiManagement.post<DBClusterPayload, any>('/DBaaS/PXCCluster/Create', toAPI(dbCluster));
   }
 
   updateDBCluster(dbCluster: DBCluster): Promise<void | DBClusterPayload> {
-    return apiManagement.post<DBClusterPayload, any>('/DBaaS/XtraDBCluster/Update', toAPI(dbCluster));
+    return apiManagement.post<DBClusterPayload, any>('/DBaaS/PXCCluster/Update', toAPI(dbCluster));
   }
 
   resumeDBCluster(dbCluster: DBCluster): Promise<void | DBClusterPayload> {
-    return apiManagement.post<DBClusterPayload, any>('/DBaaS/XtraDBCluster/Update', toResumeAPI(dbCluster));
+    return apiManagement.post<DBClusterPayload, any>('/DBaaS/PXCCluster/Update', toResumeAPI(dbCluster));
   }
 
   suspendDBCluster(dbCluster: DBCluster): Promise<void | DBClusterPayload> {
-    return apiManagement.post<DBClusterPayload, any>('/DBaaS/XtraDBCluster/Update', toSuspendAPI(dbCluster));
+    return apiManagement.post<DBClusterPayload, any>('/DBaaS/PXCCluster/Update', toSuspendAPI(dbCluster));
   }
 
   deleteDBClusters(dbCluster: DBCluster): Promise<void> {
-    const toAPI = (cluster: DBCluster): DBClusterActionAPI => ({
-      name: cluster.clusterName,
+    const body = {
+      name: dbCluster.clusterName,
       kubernetes_cluster_name: dbCluster.kubernetesClusterName,
-    });
+      cluster_type: DBClusterType.pxc,
+    };
 
-    return apiManagement.post<any, DBClusterActionAPI>('/DBaaS/XtraDBCluster/Delete', toAPI(dbCluster));
+    return apiManagement.post<any, DBClusterActionAPI>('/DBaaS/DBClusters/Delete', body);
```

Comment:
Can we add types to any? It will be easier in feature working with a payload if we will have types of this.

Reply:
Done, thank you!